maestro-client/src/main/java/maestro/drivers/IOSDriver.kt

```diff
@@ -178,8 +183,8 @@ class IOSDriver(
     }
 
     override fun contentDescriptor(): TreeNode {
-        return when (val contentDescriptorResult = iosDevice.contentDescriptor()) {
-            is Ok -> mapHierarchy(contentDescriptorResult.value)
+        return when (val viewHierarchy = iosDevice.viewHierarchy()) {
```

Comment:
Wait why are we using this here don't we need to keep the original implementation intact?

Reply:
You are correct! Will revert this once testing is done

Reply:
Removed